lib/source/pl/core/lexer.cpp

```diff
@@ -105,6 +105,67 @@ namespace pl::core {
         return c;
     }
 
+    std::optional<Token> Lexer::parseDirectiveName(const std::string_view &identifier) {
+        auto directives = Token::Directives();
```

Comment:
This can probably be a const reference to avoid copying the entire data

Reply:
I added a bunch of const refs but that one escaped. Not anymore.